bindgen-tests/tests/expectations/tests/reference_newtype_wrapper.rs

```diff
@@ -0,0 +1,23 @@
+#![allow(dead_code, non_snake_case, non_camel_case_types, non_upper_case_globals)]
+#[repr(transparent)]
+pub struct __bindgen_marker_Reference<T: ?Sized>(*const T);
+#[repr(transparent)]
+pub struct __bindgen_marker_RValueReference<T: ?Sized>(*const T);
+unsafe extern "C" {
+    #[link_name = "\u{1}_Z17receive_referenceRKi"]
+    pub fn receive_reference(
+        input: __bindgen_marker_Reference<*const ::std::os::raw::c_int>,
```

Comment:
This feels wrong? an rvalue reference to an int should be a regular pointer to an int, not a pointer-to-pointer, which is what this would end up being.